src/integrationTest/resources/log4j2-test.properties

```diff
@@ -12,10 +11,18 @@ appender.console.filter.prerelease.regex=.+\\Qis a pre-release version of OpenSe
 appender.console.filter.prerelease.onMatch=DENY
 appender.console.filter.prerelease.onMismatch=NEUTRAL
 
-rootLogger.level = warn
+appender.capturing.type = LogCapturingAppender
+appender.capturing.name = logCapturingAppender
+
+rootLogger.level = info
 rootLogger.appenderRef.stdout.ref = consoleAppender
 
-logger.testsecconfig.name=org.opensearch.test.framework.TestSecurityConfig
+logger.testsecconfig.name = org.opensearch.test.framework.TestSecurityConfig
 logger.testsecconfig.level = info
 logger.localopensearchcluster.name=org.opensearch.test.framework.cluster.LocalOpenSearchCluster
 logger.localopensearchcluster.level = info
+
+# Logger required by test org.opensearch.security.http.JwtAuthenticationTests
+logger.httpjwtauthenticator.name = com.amazon.dlic.auth.http.jwt.HTTPJwtAuthenticator
+logger.httpjwtauthenticator.level = debug
```

Comment:
Now that we're reviewing and merging these changes, should we up the log level to reduce the build output?

Reply:
This configuration is related to conversation: https://github.com/opensearch-project/security/pull/2179#discussion_r999685387